core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -250,6 +250,10 @@ public int totalOperations() {
         return operationCounter;
     }
 
+    public long lastSyncedGlobalCheckpoint() {
```

Comment:
actually, this can be completely removed - it's a leftover from the "somewhat into the future" POC. good catch.